lamassu-remote-install/install

```diff
@@ -97,11 +99,16 @@ echo $SEED > $SEED_FILE
 
 decho "Installing latest npm package manager for node..."
 retry 3 npm -g --unsafe-perm install npm@5  >> $LOG_FILE 2>&1
-NODE_MODULES=$(npm -g root)
-NPM_BIN=$(npm -g bin)
 
 decho "Installing lamassu-server..."
-retry 3 npm -g --unsafe-perm install lamassu/lamassu-server#${1-master} >> $LOG_FILE 2>&1
+mkdir -vp ${INSTALL_DIR} >> $LOG_FILE 2>&1
+cd ${INSTALL_DIR} >> $LOG_FILE 2>&1
+retry 3 npm --unsafe-perm install --production lamassu/lamassu-server#${1-master} >> $LOG_FILE 2>&1
```

Comment:
Thinking about it, I don't think we should use npm for this. We should use git to clone the correct release from GitHub, then run npm install. This way we flatten the directory structure to ``/opt/lamassu-server/lib``. It might also make it easier to update by using git.